spec/rest/tls.test.js

```diff
@@ -0,0 +1,118 @@
+"use strict";
+
+define(['ably', 'shared_helper', 'async'], function(Ably, helper, async) {
+	var currentTime, exports = {};
```

Comment:
Sorry for the confusion, I see now that this is the error you were asking about in your next comment. To be honest I'm not sure why I did that either. Will look at it again.